recipes/default.rb

```diff
@@ -242,12 +269,11 @@ class ::Chef::Recipe
   compute-vnc-proxy
   compute-api
   compute-serial-proxy
-  network
   telemetry
   telemetry-metric
 ).each do |service|
   node.default['openstack']['bind_service']['all'][service]['host'] =
-    node['ipaddress']
+    node['osl-openstack']['bind_service']
```

Comment:
Is this more of a semantic change for clarity? I see you assigned the contents of `node[ipadress]` in the attributes file.